crates/ruff/src/rules/flake8_type_checking/rules/runtime_import_in_type_checking_block.rs

```diff
@@ -75,12 +89,45 @@ pub(crate) fn runtime_import_in_type_checking_block(
                 .is_runtime()
         })
     {
-        let diagnostic = Diagnostic::new(
+        let mut diagnostic = Diagnostic::new(
             RuntimeImportInTypeCheckingBlock {
                 full_name: full_name.to_string(),
             },
             binding.range,
         );
+
+        if checker.patch(diagnostic.kind.rule()) {
+            diagnostic.try_set_fix(|| {
+                // Step 1) Remove the import from the type-checking block.
+                let source = binding.source.unwrap();
+                let deleted: Vec<&Stmt> = checker.deletions.iter().map(Into::into).collect();
```

Comment:
Nit: I don't know if that works but I find this easier to read because it tells me to what it converts it into

```suggestion
                let deleted: Vec<_> = checker.deletions.iter().map(Stmt::from).collect();
```

I'm not sure if `Stmt::from` is the right type because I have no idea what `Into::into` should be converting into :D